### DISCUSSION TITLE
Im new and need help

### DISCUSSION THREAD

**Discussion by Torched33** [dnschneid/crouton#4604] [Q&A] [answered]
![Screenshot 2021-09-14 7 22 40 PM](https://user-images.githubusercontent.com/90738751/133360112-2a11e1b6-7903-42b4-a601-3701bf95e864.png)

/usr/local/chroots/xenial already has stuff in it, how do i delete whats in that chroots?

**Comment by ghost** [comment, marked as answer]
You should, usually, use issues for this kind of question, and also go more in depth as to what is wrong. 

The problem is that a chroot with then name of `xenial` has already been created. You can use the `-u` option to update it, if you are trying to modify it, you can use the `-n` option to give a new name, other then `xenial`, or you can delete it with the `edit-chroot` command.
